dart_io_mini_samples/example/http/adding_custom_headers.dart

```diff
@@ -8,18 +8,17 @@
 import 'dart:convert';
 import 'package:http/http.dart' as http;
 
-void main() {
+main() async {
   var url = 'https://api.github.com/users/dart-lang/repos';
-  http.get(url, headers : {'User-Agent':'Dart/1.0 (My Dart client)'})
-    .then((response) {
-      List<String> repos = JSON.decode(response.body);
-      var heading = 'Repository | Star count  | Fork count';
-      print(heading);
-      print(new List.filled(heading.length, '=').join());
-      for (var repo in repos) {
-        print("${repo['name']} | "
-              "${repo['stargazers_count']} | "
-              "${repo['forks_count']}");
-    }
-  });
+  var response = await http.get(url, headers : {'User-Agent':'Dart/1.0 (My Dart client)'});
```

Comment:
Not that that's your fault, since you didn't write that originally. :) dartfmt would catch stuff like this.